lib/widget/dropdown.dart

```diff
@@ -96,11 +96,21 @@ abstract class SelectOne extends StatefulWidget with Invokable, HasController<Se
       for (var element in values) {
         // must be of value/label pair. Maybe let user overrides later
         if (element is Map) {
-          if (element['value'] != null) {
-            entries.add(SelectOneItem(
+          if (element['iconname'] != null) {
```

Comment:
no need for if/else here if we have an IconModel. Simply set it even if null.

Reply:
okay thank you